django/contrib/admin/helpers.py

```diff
@@ -213,7 +231,13 @@ def contents(self):
                         result_repr = linebreaksbr(value)
             else:
                 if isinstance(f.remote_field, ManyToManyRel) and value is not None:
-                    result_repr = ", ".join(map(str, value.all()))
+                    result_repr = []
+                    for i, remote_obj in enumerate(value.all()):
```

Comment:
What do we need to have `enumerate()` here?

Reply:
Good catch ! That was a leftover - removed it